serviceaccounts.rb

```diff
@@ -9,12 +9,16 @@
 # The test SA key is a special case for local development, and is initialized
 # but not deleted.
 class ServiceAccountContext
+  attr_reader :service_account
 
   SERVICE_ACCOUNT_KEY_PATH = "sa-key.json"
 
   def initialize(project, service_account = nil, path = nil)
     @project = project
```

Comment:
nit: name this `project_id` or `project_number` as appropriate.

Reply:
I agree, but it's an existing issue and this will cause a large amount of downstream refactoring. Feel free to file these cleanups as a separate ticket if we're planning on keeping this repo in combination with the new devops repo.

Reply:
BTW, I've been playing with a RubyMine trial, and it's not too bad. It's got a debugger and the refactoring support is all right. Not sure I'd pay $200 for it, though.

I haven't been able to get much out of the Ruby plugin in IJ.